LION/models/LPD.py

```diff
@@ -252,3 +271,92 @@ def forward(self, g):
             f_primal = self.__primal_step(f_primal, update, primal_module)
 
         return f_primal[:, 0:1]
+    
+    @staticmethod
```

Comment:
I think I added it here because it was easier for me to test if it actually solves the problem but then forgot to actually add it to LIONmodel - sorry!